tests/unit/infrastructure/s3/s3Service.test.ts

```diff
@@ -0,0 +1,108 @@
+import { S3 } from 'aws-sdk';
+import getFromS3 from '../../../../src/infrastructure/s3/s3Service';
+
+describe('S3 Service', () => {
+  it('passes the expected key to getObject if folder is defined', async () => {
+    const mockS3 = ({} as unknown) as S3;
+    const bucket = 'bucket';
+    const folder = 'folder';
+    const certNumber = 'cert123456';
+    const vin = 'VIN2345AB';
+    const mockPromise = jest
+      .fn()
+      .mockReturnValue(Promise.resolve({ Body: 'Success!', ContentType: 'application/octet-stream' }));
+    const mockGetObject = jest.fn().mockReturnValue({ promise: mockPromise });
```

Comment:
Up to you but could have just returned the mocked Promise and didn't need to create the intermediary mockPromise object I think as  mockReturnValue already returns a mock

```js
// probably would have been the same

.mockReturnValue({promise: Promise.resolve({...})})

```